README.md

```diff
@@ -67,57 +88,101 @@ Below is a comprehensive list of available configuration properties.
 |webhook.projects.<*projectId*>.secret|N/A|Webhook secret used to validate webhook requests originating from the respective projectId|
 |webhook.projects.<*projectId*>.skipSignatureCheck|N/A|Boolean to indicate whether the signature should be validated. TODO remove in favor of empty secret.|
 
-## Running Optimizely via Docker
-Alternatively, if you have Docker installed, Optimizely Agent can be started as a container:
+## Full Stack API
+
+The core Full Stack API is implemented as a REST service configured on it's own HTTP listener port (default 8080).
+The full API specification is defined in an OpenAPI 3.0 (aka Swagger) [spec](./api/openapi-spec/openapi.yaml).
+
+Each request made into the Full Stack API must include a `X-Optimizely-SDK-Key` in the request header to
```

Comment:
Feature request! I had to download an extension to add the header to the request, but it would be great if we could add this to our own chrome extension to debug Optimizely Agent from the browser. Maybe a hack week project!

Reply:
Like an Agent Console? :)